### ISSUE TITLE
{ keys } being ignored

### ISSUE THREAD

**Issue by christianbundy** [ssbc/ssb-config#44]
I'm thinking that this change may have been introduced [here](https://github.com/ssbc/ssb-config/pull/29/files#diff-4e5f7365554c896966807a1d7c82b371R67)?

```js
const ssbConfig = require('ssb-config/inject')
const ssbKeys = require('ssb-keys')

const keys = ssbKeys.generate()
const config = ssbConfig(Math.random().toString(16), { keys })

console.log(keys.public)
console.log('vs')
console.log(config.keys.public)
```

```console
$ node example.js
cjso793P3FmXfHFE6h+Ji4qRWEhLUFoQrrcodLWLJow=.ed25519
vs
aZmUau7MXDQJFKRUswLvVZ/I0C5t0NdL9nLADQkakC4=.ed25519
```

cc: @mixmix

**Comment by dominictarr** [comment]
ssb-config never used to do that, but @mixmix moved it in because you'd always load keys anyway.
I guess it should just check wether keys is already set.

**Comment by christianbundy** [comment]
Never used to do what?

I've pushed a PR to resolves this and add a test as well, see: #45.

**Comment by dominictarr** [comment]
I mean, before that change, ssb-config never used to touch the keys at all, the keys were loaded separately

https://github.com/ssbc/ssb-server/commit/da3d0b31e2c44ceb92a6292f1653605ee64215be
